_community/proposal.md

```diff
@@ -238,10 +238,15 @@ The existing code already has unit tests so we will make use of existing Apache
 
 ## Source and Intellectual Property Submission Plan
 
-Current code is Apache 2.0 licensed and the copyright is assigned to XiaoMi.
+Most of the current code is Apache 2.0 licensed and the copyright is assigned to XiaoMi.
 If the project enters incubator, XiaoMi will transfer the source code & trademark
 ownership to ASF via a Software Grant Agreement.
 
+But due to historical issues, Pegasus was based on MIT licensed codes that were initially
```

Comment:
code is mass noun. Here is the disscussion[https://english.stackexchange.com/questions/20455/is-it-wrong-to-use-the-word-codes-in-a-programming-context]